working_with_health_tool/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+import ballerinax/health.fhir.r4;
+import healthcare_samples/carinbb_package as carinbb;
+
+listener http:Listener httpListener = new (9090);
+
+public type Patient record {
+    string id;
+    string lastName?;
+    string firstName;
+    string middleName;
+    string gender?;
+};
+
+service / on httpListener {
```

Comment:
Init `httpListener` here itself.

Reply:
fixed in [82be790](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/82be790415e7ca789008605060f62b95eded8c8a)